### DISCUSSION TITLE
testing pixijs examples in jsfiddle or anywhere else?

### DISCUSSION THREAD

**Discussion by ripmurdock** [pixijs/pixijs#9216] [Q&A]
I'm just getting started learning Pixijs, and I'd like to test some of the examples.

When I copy this example: https://pixijs.io/examples/#/filters-basic/displacement-map-flag.js

into JSFiddle: https://jsfiddle.net/ripmurdock/42h7fckx/4/

I get a black box and this error in the console: Invalid X-Frame-Options header was found when loading “https://fiddle.jshell.net/_display/?editor_console=true”: “ALLOWALL” is not a valid directive.

I'm more familiar with codepen. Codepen support said running this example was Pro feature.

What's a good way to test the Pixijs examples in JSfiddle or anywhere else?

**Comment by bigtimebuddy** [comment]
Try to use absolute paths for assets: https://jsfiddle.net/bigtimebuddy/f9wdv75e/

**Comment by ripmurdock** [comment]
Thanks.  jsfiddle allowed the absolute links to pixijs.io, but it blocked the assets I uploaded to w3spaces.com.

Is there an asset hosting site that either jsfiddle or codepen will allow for testing pixijs.io?

I was able to get the displacement filter to work on my own site, although testing lots of variations on this site is cumbersome.
https://casadeveronica.com/__2kdtkpy-z6huasdfasd_pixijs-3/

If I can figure out how to make the background transparent and how to reverse the refresh to right to left when this banner animates across the screen in the opposite direction, I'll be able to move forward with this project.

I tried adding options.backgroundAlpha(0) to every element in the flag example without luck.

I also tried reversing the displacement filter, but it's still refreshing left to right.

Thanks again!
